### PR TITLE
NodeNotHealthy prometheus alert

### PR DESCRIPTION
**Pull request by adenitiu**
**How to categorize this PR?**

/area monitoring
/kind bug

**What this PR does / why we need it**:
This PR corrects the seed name mentioned in the alert and removes the wait time of 10m, as we would want to be notified immediately when a node fulfils the expression. 
The expression checks a node is not healthy in a time-window of 30 mins, hence no need to wait for additional 10m. 
 
**Which issue(s) this PR fixes**:
-

**Special notes for your reviewer**:
/invite @rickardsjp @istvanballok 

**Release note**:

```other operator
Enhance `NodeNotHealthy` Prometheus alert to show the seed name and fire immediately.
```

### CODE REVIEW

**Comment by rickardsjp** [comment]
/assign

FYI, `/invite` doesn't seem to be doing the job

**Comment by rfranzke** [comment]
> /assign
> 
> FYI, `/invite` doesn't seem to be doing the job

Prow uses `/cc` 😉

File: pkg/component/monitoring/charts/bootstrap/aggregate-prometheus-rules/seed.rules.yaml [hunk 1]
```diff
@@ -18,12 +18,12 @@ groups:
...
+    for: 0m
     labels:
       severity: warning
       type: seed
       visibility: operator
     annotations:
       summary: A node is not healthy.
       description: >
-        Node {{$labels.node}} in seed {{$labels.name}} in landscape {{$externalLabels.landscape}} is not healthy for over 10 min.
+        Node {{$labels.node}} in seed {{$externalLabels.seed}} in landscape {{$externalLabels.landscape}} has been marked as not healthy for at least 5 times in the past 30 mins.
```
Comment (rickardsjp) [inline pkg/component/monitoring/charts/bootstrap/aggregate-prometheus-rules/seed.rules.yaml]: I think "marked" as not healthy may be a bit misleading, what do you think about the following?

```suggestion
        Node {{$labels.node}} in seed {{$externalLabels.seed}} in landscape {{$externalLabels.landscape}} was not healthy for five scrapes in the past 30 mins.
```

**Review by rickardsjp — CHANGES_REQUESTED** [review]
I can't edit you PR, but please also adjust the title and the release note with the correct name `NodeNotHealthy prometheus alert`.
What do you think about this release note:

```
Enhance NodeNotHealthy prometheus alert to show the seed name and fire immediately.
```

**Comment by adenitiu** [comment]
Thanks @rickardsjp . Suggestions makes sense. Adapted accordingly.

**Review by rickardsjp — APPROVED** [review]
/lgtm

**Comment by rfranzke** [comment]
@adenitiu pls rebase

**Review by rfranzke — COMMENTED** [review]
/lgtm
/approve

**Comment by rfranzke** [comment]
@adenitiu pls adapt unit tests...

**Comment by rfranzke** [comment]
> Adding label `do-not-merge/contains-merge-commits` because PR contains merge commits, which are not allowed in this repository. Use `git rebase` to reapply your commits on top of the target branch. Detailed instructions for doing so can be found [here](https://git.k8s.io/community/contributors/guide/github-workflow.md#4-keep-your-branch-in-sync).

@adenitiu pls fix this

**Comment by adenitiu** [comment]
Closed in favour of https://github.com/gardener/gardener/pull/9311

**Pull request closed without merging**